UITests/BraintreeDropIn_UITests.swift

```diff
@@ -884,3 +884,160 @@ class BraintreeDropIn_Error_UITests: XCTestCase {
         self.waitForElementToAppear(app.buttons.containing(existsPredicate).element(boundBy: 0))
     }
 }
+
+class BraintreeDropIn_SaveCardToggleVisibleAndOn_UITests: XCTestCase {
+
+    var app: XCUIApplication!
+
+    override func setUp() {
+        super.setUp()
+        continueAfterFailure = false
+        app = XCUIApplication()
+        app.launchArguments.append("-EnvironmentSandbox")
+        app.launchArguments.append("-ClientToken")
+        app.launchArguments.append("-ThreeDSecureDefault")
+        app.launchArguments.append("-SaveCardToggleVisible")
+        app.launch()
+        sleep(1)
+
+        self.waitForElementToBeHittable(app.buttons["Add Payment Method"])
+        app.buttons["Add Payment Method"].tap()
+
+        self.waitForElementToBeHittable(app.staticTexts["Credit or Debit Card"])
+        app.staticTexts["Credit or Debit Card"].tap()
+    }
+
+    func testDropIn_saveCardSwitch_isVisible() {
+        let elementsQuery = app.scrollViews.otherElements
+
+        let cardNumberTextField = elementsQuery.textFields["Card Number"]
+        self.waitForElementToBeHittable(cardNumberTextField)
+        cardNumberTextField.typeText("4111111111111111")
+
+        let saveCardSwitch = elementsQuery.switches["Save card"]
+        self.waitForElementToBeHittable(saveCardSwitch)
+
+        XCTAssertTrue(saveCardSwitch.exists)
+
+        guard let switchValue = saveCardSwitch.value else {
+            XCTFail()
+            return
+        }
+
+        XCTAssertEqual("1", switchValue as! String)
```

Comment:
Here's another spot where we could simplify this to:

```swift
XCTAssertEqual("1", saveCardSwitch.value as? String)
```
and delete the `guard`. Also, there seems to be a rogue empty line just below this one.